src/main/java/org/folio/circulation/domain/representations/RequestByInstanceIdRequest.java

```diff
@@ -52,6 +53,6 @@
     final UUID pickupServicePointId = getUUIDProperty(json, PICKUP_SERVICE_POINT_ID);
 
     return succeeded(new RequestByInstanceIdRequest(requestDate, requesterId, instanceId,
-        requestExpirationDate, pickupServicePointId));
+        requestExpirationDate, pickupServicePointId, json.getString("patronComments")));
```

Comment:
Can this use the JSON property fetcher instead? 

```suggestion
        requestExpirationDate, pickupServicePointId, getProperty(json, "patronComments")));
```